src/ui/QuerySuggestPreview/QuerySuggestPreview.ts

```diff
@@ -22,6 +22,21 @@ export interface IQuerySuggestPreview {
   executeQueryDelay?: number;
 }
 
+/**
+ * This component adds previews of the top query results matching the currently focused query suggestion in the search box.
+ *
+ * As such, this component only works when an [`Omnibox`]{@link Omnibox} is present in the search interface.
+ *
+ * Moreover, this component can use a [result template](https://docs.coveo.com/en/413/) in its markup configuration to render previews with.
```

Comment:
I'd remove the "Moreover, ..." part and the example that follows. The external documentation will provide examples of how to use the component.

Reply:
Perfect, removing it.